### PR TITLE
Add GitHub Student Education Package to the miscellaneous section

### LINKED ISSUE #24: Add github education package to miscellaneous resources section
Github gives students free access to a wide variety of learning resources through the github student education package after verifying one's student id. Would be useful to add a link to it in the miscellaneous resources section.

### PR DESCRIPTION
**Pull request by Aniganesh**
This PR, if approved will add a link to the [github student education package](https://education.github.com) in the miscellaneous section of the `README.md`

Closes #24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
 > The Developers India Wiki for some cool & useful resources.
 
-
```
Comment (Jarmos-san) [inline README.md:5]: Just curious was this dash added here mistakenly?

File: README.md [hunk 2]
```diff
@@ -27,186 +26,190 @@ Find below some of the best and popular resources for learning new technologies/
...
-    - [Fluent C++](https://www.fluentcpp.com/) - *Advanced*
-    - [Bartek's Coding Blog](https://www.bfilipek.com/?m=1) - *Advanced*
-    - [Kenny Kerr](https://kennykerr.ca/articles/) - *Advanced*
-    - [Sutter’s Mill](https://herbsutter.com/gotw/) - *Advanced*
-    - [thoughts on cpp](https://thoughts-on-cpp.com/) - *Advanced*
-    - [Vorbrodt's C++ Blog](https://vorbrodt.blog/) - *Advanced*
-    - [foonathan::blog()](https://foonathan.net/index.html) - *Advanced*
-    - [vector<bool>{ true, true, false };](https://vector-of-bool.github.io/) - *Advanced*
-    
+* **Websites**
```
Comment (Jarmos-san) [inline README.md:29]: Formatting issues can be taken care of in a separate PR

File: README.md [hunk 3]
```diff
@@ -250,22 +253,22 @@ Graphics abstraction:
...
+* [How HTTPS Works](https://howhttps.works/)
+* [How DNS Works](https://howdns.works/)
+* [Stanford CS Education Library](http://cslibrary.stanford.edu/)
+* [Git.WTF](https://git.wtf/)
+* [Oh shit, git!](https://ohshitgit.com/)
+* [OpenStax Textbooks](https://cnx.org/)
+* [Computer-Science-Resources](https://github.com/the-akira/Computer-Science-Resources.git)
+* [every-programmer-should-know](https://github.com/mtdvio/every-programmer-should-know)
+* [97 Things Every Programmer Should Know](https://97-things-every-x-should-know.gitbooks.io/97-things-every-programmer-should-know/en/)
+* [Github Student Education package](https://education.github.com)
```
Comment (Jarmos-san) [inline README.md:265]: Looks good enough but generally speaking PRs should be specific. And this one touched upon way too much stuff than what you described in the PR.

**Review by Jarmos-san — CHANGES_REQUESTED** [review]
Looks good but could you just add the link to the GitHub Education thing & leave out the rest of changes you made as it is for now (see the other comments for reasons)? You can open another PR describing some of the formatting issues in the `.md` file.

**Comment by Bhupesh-V** [comment]
@Aniganesh Feel free to raise a new PR or address the comments in this one and let us know if you want it reviewed. Closing this for now.

**Pull request closed without merging**